rules/windows/lateral_movement_remote_file_copy_hidden_share.toml

```
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/11/04"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/04"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies remote file copy attempt to a hidden network share. This may indicate lateral movement or data staging
+activity.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Remote File Copy to a Hidden Share"
+risk_score = 47
+rule_id = "fa01341d-6662-426b-9d0c-6d81e33c8a9d"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+process.name in ("cmd.exe","powershell.exe","robocopy.exe","xcopy.exe") and wildcard(process.command_line,"*copy*$*","*move*$*")
```

I think it would be a little more efficient to break that `command_line` into `process.args` since it would only rely on one wildcard (though I know ordering isn't necessarily preserved)

```suggestion
process where event.type in ("start", "process_started") and
  process.name : ("cmd.exe", "powershell.exe", "robocopy.exe", "xcopy.exe") and 
  process.args : ("copy", "move") and process.args : "$*"
```

---

the copy and move args conversion (from cmdline to args) is OK, but the $ sign of hidden share will be always in between strings (e.g. \\abc\c$\somewhere\...)